### PR TITLE
Fixes and some improvements phase 2

### PR DESCRIPTION
**Pull request by marcelpujol**

### CODE REVIEW

**Review by Xelit3 — DISMISSED** [review]
Approved mate @marcelpujol

**Comment by Xelit3** [comment]
This PR will not create a new package since no changes in package.json are done, is this corect @marcelpujol ?

File: src/components/object-counter/object-counter.tsx [hunk 1]
```diff
@@ -190,5 +190,11 @@ export class ObjectCounter {
   private setReadonly(): void {
     this.readonly = this.field?.readOnly;
   }
+
+  private isFilled(): boolean {
+    return (this.field.value) 
+      ? this.field.value?.counted !== null || this.field.value?.image
```
Comment (javierpardollama) [inline src/components/object-counter/object-counter.tsx:196]: Not sure about that Condition. Should not the folowing do the same?
```
return (this.field.value) ? this.field.value?.counted || this.field.value?.image;

**Review by javierpardollama — APPROVED** [review]

File: src/components/object-counter/object-counter.tsx [hunk 2]
```diff
@@ -190,5 +190,11 @@ export class ObjectCounter {
   private setReadonly(): void {
     this.readonly = this.field?.readOnly;
   }
+
+  private isFilled(): boolean {
+    return (this.field.value)
```
Comment (javierpardollama) [inline src/components/object-counter/object-counter.tsx:195]: Not sure about that Condition. Should not the folowing do the same?
```
return (this.field.value) ? this.field.value?.counted || this.field.value?.image;

**Review by javierpardollama — CHANGES_REQUESTED** [review]
Made some Comments. Can you check them please?

**Comment by marcelpujol** [comment]
@javierpardollama To me it is correct, because in some cases the value could be filled without image entering the counted value in a manual way. We can do reflexion if you want, but to me it is clear as it is.

**Review by javierpardollama — APPROVED** [review]

**Comment by javierpardollama** [comment]
> @javierpardollama To me it is correct, because in some cases the value could be filled without image entering the counted value in a manual way. We can do reflexion if you want, but to me it is clear as it is.

Perfect then. No more claims from my end. You got my approval

**Review by RocioDelJ — APPROVED** [review]

**Pull request merged**